SettingsList.py

```diff
@@ -2430,22 +2430,16 @@ def __init__(self, name, gui_text, min, max, default, step=1,
             True: {'settings': ['shuffle_weird_egg']},
         },
     ),
-    Setting_Info(
-        name           = 'dungeon_shortcuts',
-        type           = list,
-        gui_text       = 'Dungeon Boss Shortcuts',
-        gui_type       = "MultipleSelect",
+    Combobox(
+        name           = 'dungeon_shortcuts_choice',
+        gui_text       = 'Dungeon Boss Shortcuts Choice',
+        default        = 'off',
         choices        = {
-            'deku_tree': 'Deku Tree',
-            'dodongos_cavern': 'Dodongo\'s Cavern',
-            'jabu_jabus_belly': 'Jabu Jabu\'s Belly',
-            'forest_temple': 'Forest Temple',
-            'fire_temple': 'Fire Temple',
-            'water_temple': 'Water Temple', # doesn't do anything, but added to prevent confusion
-            'shadow_temple': 'Shadow Temple',
-            'spirit_temple': 'Spirit Temple'
+            'off':       'Off',
+            'choice':    'Choice dungeons',
```

Comment:
Should be "Choose dungeons"

Reply:
Fixed : https://github.com/TestRunnerSRL/OoT-Randomizer/pull/1493/commits/a0e2952eabb59bda65bd4ee3ca66f82250fcf08a